01-js-intro-and-jquery/app.js

```diff
@@ -0,0 +1,27 @@
+var main = function() {
+    
+    $('.hidden').fadeIn(5000, function() {
+
+        $( '.alias' ).focus()
+    });
+
+    $('#btn').click( function( event ) {
+
+        $.get("http://bootcamp.aws.af.cm/welcome/yourname", function(text) {
```

Comment:
This is not a correction, but more like a suggestion... :P
It makes more sense to me (talking 'bout concepts...) to call your response ``data`` or ``response`` instead ``text``, since you could get any kind of data (objects, arrays, strings, etc.). But as I say, it's not incorrect, just thinking aloud!